drivers/gpu/drm/i915/intel_initial_modeset.c

```diff
@@ -80,11 +80,8 @@ static bool use_connector(struct drm_connector *connector)
 			break;
 		}
 
-		connector->status = connector->funcs->detect(connector, true);
```

Comment:
How about using the below for the use_connector?
> if (connector->funcs && conector->funcs->detect)
        status = connector->funcs->detect(connector, true);
   else
        status = connector_status_connected;